### PR TITLE
Client mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/client/network/ClientAdvancementManager.mapping [hunk 1]
```diff
@@ -1,8 +1,15 @@
 CLASS dgy net/minecraft/client/network/ClientAdvancementManager
+	CLASS net/minecraft/client/network/ClientAdvancementManager$a Listener
+		METHOD a setProgress (Lq;Ls;)V
+		METHOD e selectTab (Lq;)V
 	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD b client Lcvb;
 	FIELD c manager Lr;
+	FIELD d advancementProgresses Ljava/util/Map;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/network/ClientAdvancementManager.mapping:8]: advancementProgresses vs advancementProgressMap? What is it keyed by?

File: mappings/net/minecraft/client/render/model/ModelLoader.mapping [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 	CLASS dvq$a ModelLoaderException
-	FIELD A BLOCK_STATE_FACTORY_OVERRIDES Ljava/util/Map;
+	FIELD A STATIC_DEFINITIONS Ljava/util/Map;
 	FIELD B resourceManager Lxd;
 	FIELD C spriteAtlas Ldty;
 	FIELD D modelsToLoad Ljava/util/Set;
 	FIELD E variantMapDeserializationContext Ldle$a;
 	FIELD F unbakedModels Ljava/util/Map;
-	FIELD G modelRotationCache Ljava/util/Map;
+	FIELD G bakedModelCache Ljava/util/Map;
```
Comment (asiekierka) [inline mappings/net/minecraft/client/render/model/ModelLoader.mapping:9]: I think, if it's still rotation-keyed, it's important to keep that in mind.

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/client/render/model/ModelLoader.mapping (hunk 2 above)
Comment (Runemoro) [reply mappings/net/minecraft/client/render/model/ModelLoader.mapping:9]: It's keyed by `ModelBakeSettings`, which used to be `ModelRotationContainer`, but actually contains more than just rotation (for example `uvLock`)

Comment (asiekierka) [reply mappings/net/minecraft/client/render/model/ModelLoader.mapping:9]: OK

File: mappings/net/minecraft/client/network/ClientAdvancementManager.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/client/network/ClientAdvancementManager.mapping:8]: It's an Advancement -> AdvancementProgress map.

Comment (asiekierka) [reply mappings/net/minecraft/client/network/ClientAdvancementManager.mapping:8]: Right.

**Pull request merged**